build/caddy/build.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/bash
+#
+# {{{ CDDL HEADER
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+# }}}
+
+# Copyright 2024 Guo-Rong Koh
+
+. ../../lib/build.sh
+
+PROG=caddy
+VER=2.7.6
+PKG=ooce/server/caddy
+SUMMARY="Caddy web server"
+DESC="Fast and extensible multi-platform HTTP/1-2-3 web server with automatic \
+HTTPS"
+
+CONFIG=etc/${PREFIX#/}/$PROG
+DATA=var/${PREFIX#/}/$PROG
+
+set_arch amd64
+set_gover
+
+XFORM_ARGS="
+    -DPREFIX=${PREFIX#/}
+    -DPROG=$PROG
+    -DVERSION=$VER
+    -DsVERSION=
```

Comment:
Given that you set this empty, i.e. the service is not versioned, you might as well drop this parameter and remove it in the manifest and local.mog

Reply:
Done.